### PR TITLE
[WIP] Revamping "Ian's Adventure" into a station job trait

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
I remembered making [this](https://github.com/tgstation/tgstation/pull/62617) PR roughly two years ago and having to convince oranges to merge it because he's rightfully a bit skeptical. Moving the dog around thru dchat is tedious. The most fun we've had is slooooowly dragging Ian to the engineering department and have him feed himelf to the SM (as well as a few more folks tried to save a mindless puppy) but not much else... beside doing it twice because of the multiple lives component.

I promised I'd actually improve it later, though between inertia and unwillingness I let two years pass by.

Back to present day: my plan here is to make Ian* player-controllable, while letting ghosts backseat and provide a guidance of some sort.

*or in the unfortunate case he isn't around or something, either Lisa or a random corgi with a random-ass dog name.

Because It's a WIP, here are some things that I still have or would like to do:
- [ ] Alert icons for the various observer-given buffs to show to Ian, with timers showing their remaining duration. MOSTLY DONE
- [ ] Add a click-drag ability that lets the player squeeze through airlocks to the other side. MOSTLY DONE
- [ ] Add the ability to summon a hat from a fixed pool if they're wearing none.
- [ ] Add UI slots for mouth-held items and the hat (pondering about the back slot, it wouldn't be easy for a dog to take it off on its own tbf)
- [ ] Better observer-Ian interactions, while keeping the 'spirit medium' vibes as low as possible.
- [ ] An achievement for surviving all the way to CC without dying once. MOSTLY DONE

## Why It's Good For The Game
This is one of those station traits that are at the end quite meager, and could frankly use some extra beef.

## Changelog

:cl:
add: Revamped "Ian's Adventure" into a station job trait. Play as Ian (or a backup corgi, if somehow unavailable), or guide him as an observer.
qol: dogs can now consume jumbo dog bone treats on their own.
balance: Only one Lisa (The female counterpart of Ian, from the corgi livestock crate) can exist at a time.
/:cl:

### CODE REVIEW

File: code/modules/spells/spell_types/self/voice_of_god.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	/// The modifier put onto the power of the command
 	var/power_mod = 1
 	/// A list of spans to apply to commands given
-	var/list/spans = list("colossus", "yell")
+	var/list/spans = list(SPAN_YELL, SPAN_YELL)
```
Comment (Profakos) [inline code/modules/spells/spell_types/self/voice_of_god.dm:21]: Accidentally missed a SPAN_COLOSSUS...

**Comment by Profakos** [comment]
Finally Ian can become a true adventurer.

**Comment by Jacquerel** [comment]
Slow though it may be I have to admit that I still find "twitch plays Ian" more charming than "one guy plays Ian and ignores a bunch of ghosts he has no incentive to listen to".
Conversely I also don't think observers being able to make Ian slightly more durable or atmospheric-resistant is going to be all that exciting, the former because he has fuck all hp anyway and the latter being especially problematic because it's just going to run out at the wrong time.

Would this also leave Ian as able to converse in spoken common with random people? That's a side effect that I will say I don't much enjoy about making animal sapient, maybe we should move language granting to the sapience potion instead...

**Comment by Ghommie** [comment]
Those are all good points and overall a major reason why this is still a WIP, except for the ability to speak common, which is something I haven't thought much about but that'd be a nice communication barrier between the ghosts, Ian and the rest of the crew. Good ideas will come with time.

**Comment by EricZilla** [comment]
Okay but Ian being able to hear deadchat, and then trying to act on it but not being understood by the humans, is kinda funny

"Whats that Ian? The janitor is trapped in the void?" dog leads the captain to a random heretic rune

**Comment by Ghommie** [comment]
Putting this on hold for a moment. There's a rework on how some mobs, like parrots, mooks and any mob with the "fetch" pet command, can carry their items that I have to do first, and it's cooking slower than I thought.

**Pull request closed without merging**